Jenkinsfile

```diff
@@ -41,13 +41,10 @@
 //@Library(value="pipeline-lib@your_branch") _
 
 def doc_only_change() {
-    def rc = sh script: 'if [ "' + env.CHANGE_ID + '''" = "null" ]; then
-                              mb_modifier="^"
-                         fi
-                         git diff-tree --no-commit-id --name-only \
-                           $(git merge-base origin/''' + target_branch +
-                      '''$mb_modifier HEAD) HEAD | \
-                           grep -v -e "^doc$"''',
+    def rc = sh label: "Determine if doc-only change",
+                script: "CHANGE_ID=${env.CHANGE_ID} " +
+                        "target_branch=${target_branch} " +
+                        'ci/doc_only_change.sh',
                 returnStatus: true
```

Comment:
CHANGE_ID is already inherited to the script so does not need to be set.
target_branch comes from either CHANGE_TARGET or BRANCH_NAME, both of which are visible to the script.

Reply:
> CHANGE_ID is already inherited to the script so does not need to be set.

While it's  true that some things will be exported so that the script will have them in it's environment already, I think explicitly setting them is useful to document external values that the script needs.

> target_branch comes from either CHANGE_TARGET or BRANCH_NAME, both of which are visible to the script.

￼True.  But since `Jenkinsfile` has already figured it out, seems easiest and most consistent to just pass it to the script.  Ditto for documenting the external values.